### PR TITLE
Sets runtime timestamp in Postgres to `PST`

### PR DESCRIPTION
**Pull request by amanji**

### CODE REVIEW

**Review by WadeBarnes — APPROVED** [review]

**Comment by WadeBarnes** [comment]
@ianco, please have a look before we merge.

**Comment by WadeBarnes** [comment]
I confirmed we'd be looking into the future if we did not set the timezone first; https://github.com/bcgov/von-bc-registries-agent/pull/288#issuecomment-898628135

File: data-pipeline/bcreg/bcregistries.py [hunk 1]
```diff
@@ -741,6 +742,7 @@ def get_max_date_before_now(self):
         try:
             # create a cursor
             cur = self.conn.cursor()
+            cur.execute("""SET TIME ZONE """ + BC_REGISTRIES_TIMEZONE)
```
Comment (WadeBarnes) [inline data-pipeline/bcreg/bcregistries.py]: I'd recommend switching to using the cursor inside a `with` block so `.close()` is calls automatically and not explicitly.  That way the session and the time-zone setting are finite even if there is an error.

Comment (amanji) [reply data-pipeline/bcreg/bcregistries.py]: Good point

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**